### ISSUE TITLE
Callback function not supported with scroll spy enabled

### ISSUE THREAD

**Issue by rftb** [inorganik/countUp.js#302]
```
window.onload = function () {
	var numAnim = new countUp.CountUp("cumulativehours", 676357.741, {
		enableScrollSpy: true,
		decimalPlaces: 3,
		duration: 2,
		decimal: ".",
		separator: ",",
	});

	numAnim.start(() => console.log('done'));
};
```

**Comment by inorganik** [comment]
The umd script is the same code, but when you enable scroll spy it doesn't support the callback.

In your case, what's happening is countUp gets constructed, and you immediately call start, setting a callback. But when the scroll handler function finally calls start, it does so with no arguments effectively overwriting the callback you set before to nothing.

If you needed to support the callback for when scroll spy enabled, I could try and support that.

**Comment by rftb** [comment]
I see, that makes sense. It definitely would be a nice feature.

Case: I have a ticker below the fold and I don't want that ticker to begin animating until the element is visible in the viewport (perfect scrollSpy satisfied that case). Then, I need to continue incrementing that initial animation with some custom code (in the callback function), where I met a limitation with the library in its current state.

Also, a big thank you for replying so promptly. Neat library and easy to use otherwise!

**Comment by inorganik** [comment]
fixed in version 2.5.0
